### PR TITLE
Minor fix of GeoJSON and OSMXML support added

### PR DESCRIPTION
**Pull request by mazlixek**
description in commits

### CODE REVIEW

File: gwt-ol3-client/src/main/java/ol/format/OSMXMLFeatureOptions.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     * @param projection
+     */
+    @JsProperty
+    public native void setFeatureProjection(Projection projection);
+
+    /**
+     * @param rightHanded
+     */
+    @JsProperty
+    public native void setRightHanded(boolean rightHanded);
```
Comment (TDesjardins) [inline gwt-ol3-client/src/main/java/ol/format/OSMXMLFeatureOptions.java]: There is no option 'rightHanded' in OsmXml.

**Comment by TDesjardins** [comment]
Could you also create a testcase for this format?

**Comment by mazlixek** [comment]
I will make the change and I will try to create the test case... based on GeoJSONTest, although there is no writeFeature support in OSMXML so I guess I will have to use a simple string for it.

**Comment by TDesjardins** [comment]
I think a simple string is sufficiant for test data.

**Comment by mazlixek** [comment]
I removed the setRightHanded function and created a test case, I just don't know how to launch the tests so I don't know if it works...

**Comment by TDesjardins** [comment]
The tests launch every time you build with maven:

https://travis-ci.org/TDesjardins/gwt-ol3/builds/220496211#L445

Last step you have to do is to add your test class to the testsuite:

https://github.com/TDesjardins/gwt-ol3/blob/master/gwt-ol3-client/src/test/java/ol/GwtOL3TestSuite.java#L95

**Comment by mazlixek** [comment]
I actually had to use -DskipTests everytime I was building, since the Test didn't work for me. I found out it is due to a corporate proxy... even though I have cntlm, and maven build works, the tests for some reason do not. Anyway I just tried it with mobile hotspot and it worked.

**Comment by TDesjardins** [comment]
Managed! Thanks!

**Comment by TDesjardins** [comment]
This fixes #76 .

**Pull request merged**